### ISSUE TITLE
In the `src/test/java` directory, replace "Paths.get(String)" to "java.nio.file.FileSystems.getDefault().getPath(String)"

### ISSUE THREAD

**Issue by kazurayam** [kazurayam/materialstore#442]
I should not use `java.nio.file.Paths.get(String path)`. I should prefer `java.nio.file.FileSystem`. 

In the  `src/test/java` directory, there are 99 matches of the `Paths` string in 39 files.

In the `src/test/java` directory, rewrite it to "java.nio.file.FileSystems.getDefault().getPath(String)"

**Comment by kazurayam** [comment]
No, I would not do this. 

In the branch `issue442` I modified nearly 20 .java files and realized that replacing `java.nio.file.Paths.get(String path)` to `java.nio.file.FileSystems.getDefault().getPath(String path)` is meaningless.

In #440 I needed to avoid `Paths.get(String)` in the `src/main/java` files in order to support S3 FileSystem in future. That's meaningful.

However, In the  `src/test/java` files, `Paths.get()` is suffient.
